go/tools/builders/replicate.go

```diff
@@ -30,13 +30,15 @@ const (
 	softlinkMode
 )
 
-type replicateOption func(*replicateConfig)
-type replicateConfig struct {
-	removeFirst bool
-	fileMode    replicateMode
-	dirMode     replicateMode
-	paths       []string
-}
+type (
```

Comment:
It looks like this repo uses gofumpt, set in .vscode/settings.json. I was surprised it made this change, but maybe it's intentional? 🤷